packages/apple-pay-integration/src/apple-pay-button-strategy.ts

```diff
@@ -125,6 +126,12 @@ export default class ApplePayButtonStrategy implements CheckoutButtonStrategy {
 
         assertApplePayWindow(window);
 
+        if (!this._sessionFactory.canMakePayment()) {
+            throw new PaymentMethodFailedError(
+                'This device is not capable of making Apple Pay payments',
```

Comment:
The message that could be shown for customer should be translated. I think it might be great to add console.error before throwing an error, and then throw an error with specific key via onError callback (if it is available)

Reply:
We do not need to show this error to customer. Otherwise we will be faced with UX issues that can confuse customer a bit.